dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostAPIImpl.java

```diff
@@ -84,8 +84,10 @@ public Host findDefaultHost(User user, boolean respectFrontendRoles) throws DotS
 
             APILocator.getPermissionAPI().checkPermission(host, PermissionLevel.READ, user);
             return host;
-        } catch (Exception e) {
 
+        } catch (DotSecurityException | DotDataException e) {
+            throw e;
+        } catch (Exception e) {
             throw new DotRuntimeException(e.getMessage(), e);
         }
```

Comment:
this method have DotSecurityException and DotDataException in the method signature but really it never thrown any of them

Reply:
log message?